examples/cymbal-bank/5-appinfra/accounts-contacts/envs/shared/variables.tf

```diff
@@ -14,25 +14,24 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "CI/CD project ID"
-  type        = string
-}
-
 variable "region" {
   description = "CI/CD region"
   type        = string
-}
-
-variable "env_cluster_membership_ids" {
-  description = "Cluster Membership IDs"
-  type = map(object({
-    cluster_membership_ids = list(string)
-  }))
+  default     = "us-central1"
 }
 
 variable "buckets_force_destroy" {
   description = "When deleting the bucket for storing CICD artifacts, this boolean option will delete all contained objects. If false, Terraform will fail to delete buckets which contain objects."
   type        = bool
   default     = false
 }
+
+variable "remote_state_bucket" {
+  description = "Backend bucket to load Terraform Remote State Data from previous steps."
+  type        = string
+}
+
+variable "envs" {
```

Comment:
I suggest renaming this variable, to avoid confusion with the `envs` variable that is defined throughout the repo:
https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/1-bootstrap/variables.tf#L52
https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/test/setup/outputs.tf#L26

Reply:
maybe something like, `read_from_envs`